ledbuffer.h

```diff
@@ -63,7 +63,7 @@ class LEDBuffer
 
   private:
 
-    std::vector<CRGB> _leds;
+    std::deque<CRGB> _leds;

```

Comment:
nit: deque may be better semantically, but also requires more allocations.

if perf and memory use are critical here, vector may still be the best option, as long as you append from the end (moving will be just memmove which is usually cheap). Or you could do a ring buffer.

But most likely it won't be an issue.